services/blockchain-connector/methods/controllers/peers.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const peerInfo = require('../../src/sdk_v5/peers');
+
+const getPeers = async params => {
+	const response = await peerInfo.getPeers(params);
+
+	return {
+		data: response.data,
+		meta: response.meta,
+	};
```

Comment:
Let us not use the standard response envelope here. Kindly return the response directly as it is right now for all the remaining methods.
The standard response envelope can be part of the microservice that serves the user requests.